cmd/root.go

```diff
@@ -45,9 +45,10 @@ type Cli struct {
 func NewCli() *Cli {
 	cli := &Cli{
 		rootCmd: &cobra.Command{
-			Use:   "kubecm",
-			Short: "KubeConfig Manager.",
-			Long:  printLogo(),
+			Use:     "kubecm",
+			Short:   "KubeConfig Manager.",
+			Long:    printLogo(),
+			Aliases: []string{"kc"},
```

Comment:
Aliases in root are useless because all code is packaged into a binary executable file, which is actually executed by the system.

This can be done by adding `alias kc='kubecm'` to `.zshrc`.

Reply:
OK, updated.